source/math/FloatMatrix.ooc

```diff
@@ -141,30 +145,31 @@ FloatMatrix : cover {
 		result := 0.0f
 		for (i in 0 .. this height)
 			result += this[i, i]
+		this free(Owner Callee)
```

Comment:
Because thats the convention. All method calls and operators have to free all arguments including this that have the owner set to callee.